src/pages/account/AccountIndex.js

```diff
@@ -1,29 +1,54 @@
 import React, { useState, useEffect } from "react";
-
+import { Redirect } from "react-router-dom";
+import styled from "styled-components";
 import AxioxApi from "../../utils/AxiosApi";
 import NavBar from "../../components/NavBar";
+import Card from "../../components/Card";
 
 const AccountIndex = () => {
   const [myChildren, setMyChildren] = useState([]);
+  const [redirect, setRedirect] = useState(false)
 
   useEffect(() => {
     const fetchUserData = async () => {
       const result = await AxioxApi("myChildren");
       const usersChildren = await result.data.children;
```

Comment:
I thought you had to wait for the result.data.children to be in the usersChildren variable and after that you can use it @michaelkoek is this necessary?

Reply:
I understand Mohamuds response it's because Axios already does it. It doesn't do any harm to leave (just leave it, tho, it's nicer, imo) but when you use `fetch` than you would need it

Reply:
Okay, thats new for me. I didnt know that..

Reply:
Actually, after thinking about it (multitasking is not my best feature), Sophie is right, you should await the `result`.
I was confused with the fact that you don't have to to convert it to/use `.json()`; that's what Axios does automatically.

But you have the await the result of the axios response.